app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_sep = command.split()
+    if command_sep[0] == "mv":
+        old_name = command_sep[1]
+        new_name = command_sep[-1].split("/")[-1]
+        directory = command_sep[-1].split("/")
```

Comment:
reimplement this logic, since such hardcode is not good. Remember that you can `split()` like this:
`var, var1, var2 = some_string.split()`